### PR TITLE
strip out unicode characters when registering exams with Software Secure

### PR DESCRIPTION
**Pull request by ibrahimahmed443**

### CODE REVIEW

**Comment by chrisndodge** [comment]
@afzaledx @muhhshoaib

File: edx_proctoring/backends/tests/test_software_secure.py [hunk 1]
```diff
@@ -266,6 +266,52 @@ def assert_get_payload_mock_no_policy(exam, context):
                     attempt = get_exam_attempt_by_id(attempt_id)
                     self.assertIsNone(attempt['review_policy_id'])
 
+    def test_attempt_with_unicode_characters(self):
+        """
+        test that the unicode characters are removed from exam names before registering with
+        software secure.
+        """
+
+        def is_ascii(s):
```
Comment (muhhshoaib) [inline edx_proctoring/backends/tests/test_software_secure.py]: rename the parameter name

**Comment by muhhshoaib** [comment]
Just a small change. Otherwise looks great :+1:

**Comment by chrisndodge** [comment]
+1

**Pull request merged**